contracts/dapis/proxies/interfaces/IDapiProxy.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../interfaces/IDapiServer.sol";
+
+interface IDapiProxy {
+    function read() external view returns (uint224 value, uint32 timestamp);
```

Comment:
Should we also add the 2 getter functions for the public properties here (`dapiServer`, `dataFeedId`)?

Reply:
Tru